cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -357,3 +357,20 @@ body depth_search cfe_internal_docroot_application_perms
       depth => "inf";
       exclude_dirs => { "logs" };
 }
+
+############################################################################
+
+body perms state_dir_perms
```

Comment:
feels redundant to say perms twice in the same line.

Reply:
Suggestions? I am working on another PR now for master first since we decided to unblock this from the release train.